src/app/application/components/route-view/RouteView.vue

```diff
@@ -16,29 +16,15 @@
       :env="env"
       :can="can"
       :route="{
-        update: (params: Record<string, string | undefined>) => {
```

Comment:
**Issue**: (thread unrelated to line of code) Reloading a list view while on page >1 doesn’t work correctly. It always loads page 1.

Reply:
I added a fix and a couple more e2e tests to cover this, lmk!